en-us/give-up-optifine.md

```diff
@@ -54,6 +54,15 @@ Here is the list of OptiFine features that are supported in this modpack:
 
 ² Snow is opt-in via settings as it changes some textures too, making it weird with some resource packs.
 
+³ The mod Zoomify is used to get the zoom function inside the modpack. The configuration of the mod provided by the modpack is not identical to the OptiFine zoom since most people prefer a smooth than an instant zoom. The following are instructions on how to customize the mod in order to make zooming identical to OptiFine:
```

Comment:
The ³ doesn't refer to anything

Reply:
It still doesn't refer to anything. See how I've done it with the other superscript numbers.

Reply:
If you look on my fork / patch-1, you can see that I don't have the same footer as the one before. I merged the pull request that you did and now it's different to the one before.

Reply:
That's true, but I'm not referring to that line. I mean that line 27 lacks the `³`, I just cannot comment on lines you haven't edited.

Though now that I look at it, it should even be `²` due to the order and `³` should be the better snow message.

Reply:
I've added ³ to line 27, anything else hat needs to be chnaged / done?

Reply:
Make it a ² because of the order and make better snow the ³.

Reply:
Done!